src/google-logger.mjs

```diff
@@ -41,6 +45,13 @@ export class GoogleLogger {
       source = classifyAcquisition(source, true);
     }
 
+    if (checkpoint === 'audiences') {
+      source = anonymizeAudience(source, target);
+      if (source == null) {
+        return;
+      }
+    }
+
```

Comment:
```suggestion

```

Reply:
Oops… forgot to remove that when I moved the logic over 😓